packages/usa-radio/src/usa-radio.stories.js

```diff
@@ -38,3 +43,17 @@ export const TileAriaDisabled = TileTemplate.bind({});
 TileAriaDisabled.args = {
   disabled_state: "aria-disabled",
 };
+
+export const Indeterminate = Template.bind({});
+Indeterminate.argTypes = {
+  indeterminate: {
+    defaultValue: true,
+  },
+};
+
+export const IndeterminateTile = TileTemplate.bind({});
+IndeterminateTile.argTypes = {
+  indeterminate: {
+    defaultValue: true,
+  },
+};
```

Comment:
## Indeterminate radio stories

Added indeterminate radio stories since it was easy to do so using new control. 

I chose to not deactivate the control so that it's easy to reset after a selection is made!